app/src/app/components/sidebar/charts/HorizontalBarChart.tsx

```diff
@@ -39,15 +39,17 @@ export const HorizontalBar = () => {
           width={500}
           data={mapMetrics.data}
```

Comment:
Q: Do you have a sense of how we can order the data with recharts? From what I can tell by default it puts new values first. Ordering by zone would be a nice to have at this stage and required I'd expect eventually.